todo_app/app.py

```diff
@@ -1,11 +1,32 @@
 from flask import Flask
+from flask import render_template, request, redirect, url_for
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items
+from todo_app.data.session_items import add_item
+
 
 app = Flask(__name__)
-app.config.from_object(Config())
+app.config.from_object(Config)
```

Comment:
Is there a reason you've reverted this change? This seems to break your app to me, since the `SECRET_KEY` doesn't get set correctly!
```suggestion
app.config.from_object(Config())
```